### ISSUE TITLE
Include content as meta data text representation

### ISSUE THREAD

**Issue by marcorivm** [chubin/cheat.sh#28]
By having a text  representation inside the meta description tag on the html header we will enable clients like telegram that leverage that info to create link previews to give more info to the users at a glance.

**Comment by chubin** [comment]
Could you please provide an example of such a description? Just to understand you better.

**Comment by marcorivm** [comment]
For example if you look at this medium article https://medium.com/@Pier/vue-js-the-good-the-meh-and-the-ugly-82800bbe6684 and look at the source code you would notice several meta fields mainly the title and description meta fields

![image](https://user-images.githubusercontent.com/1222598/42349343-18d7403e-8072-11e8-93e0-384bfb32aa20.png)

These are used to create the content previews when posting links to facebook and other platforms

By including this, Cheat.sh will be able to  show a preview with the answer

**Comment by chubin** [comment]
@marcorivm Ok, you mean OpenGraph. Could you please check whether wttr.in works for you? We have Opegraph data there.

https://es.wttr.in/Mexico

**Comment by marcorivm** [comment]
Yeah I was initially referring to just the <meta name="description"> tag but opengraph would be nice too, opengraph would be the more modern option to just the meta description tag.

Wttr.in seems to work good but you are not including og:description tag so it won't show any more data than the title
![image](https://user-images.githubusercontent.com/1222598/42352793-47d58b92-8082-11e8-8cef-bd48c149c4bd.png)
